typescript/best-practice-web/test/pages/cart.page.ts

```diff
@@ -0,0 +1,15 @@
+import Page from './page';
+
+class CartPage extends Page {
+    private _urlPart:string = 'cart.html';
```

Comment:
You have all the pages in the `e2eConstants.ts`, in the const `PAGES`, can you or use the constant, or remove the constant. Now you have it in multiple places and that is not a good practive 😉